docs/newdocs/scala_versions.md

```diff
@@ -19,64 +19,58 @@ unused/undeclared dependency checking and test code coverage checking) via the
 repository is mapped to `rules_scala_annex`:
 
 */BUILD.bazel*
-```python
+
+```starlark
 load(
     "@rules_scala_annex//rules/register_toolchain.bzl",
     "register_bootstrap_toolchain",
     "register_zinc_toolchain",
 )
 
+
+# You'll need to pull these in via `rules_jvm_external`
 compiler_classpath_2_13 = [
-    "@scala_compiler_2_13//jar",
-    "@scala_library_2_13//jar",
-    "@scala_reflect_2_13//jar",
+    "@annex//:org_scala_lang_scala_compiler",
```

Comment:
Do we want these to be `@annex` or should we assume that the person setting up the Scala toolchain has their own maven dependency tree?

Reply:
Oh, that's a good point. The comment above says "You'll need to pull these in via `rules_jvm_external`", but it would be helpful to mention that `annex` should be replaced with the name of their dependency tree. I'll do that.

Reply:
I suggest you rename it something like `@maven` I worry that people will miss your note and be confused as to how this is supposed to come from annex. I think both the `@maven` and your note will hopefully avoid that confusion.

Reply:
Good idea. I'll rename it.